src/quickgui/qgsquickutils.cpp

```diff
@@ -32,7 +34,40 @@ QgsQuickUtils::QgsQuickUtils( QObject *parent )
 {
 }
 
-double QgsQuickUtils::screenUnitsToMeters( QgsQuickMapSettings *mapSettings, int baseLengthPixels ) const
+/**
+ * Makes QgsCoordinateReferenceSystem::fromEpsgId accessible for QML components
+ */
+QgsCoordinateReferenceSystem QgsQuickUtils::coordinateReferenceSystemFromEpsgId( long epsg )
+{
+  return QgsCoordinateReferenceSystem::fromEpsgId( epsg );
+}
+
+QgsPointXY QgsQuickUtils::pointXYFactory( double x, double y ) const
```

Comment:
Can those be renamed to `QgsQuickUtils::pointXY(x,y)`, to match the naming schema `Qt.point(x, y)`?